domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/optionalAssertions.kt

```diff
@@ -3,12 +3,25 @@
 package ch.tutteli.atrium.domain.robstoll.lib.creating
 
 import ch.tutteli.atrium.assertions.Assertion
+import ch.tutteli.atrium.core.Option
 import ch.tutteli.atrium.creating.Expect
 import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.creating.changers.ExtractedFeaturePostStep
 import ch.tutteli.atrium.reporting.RawString
 import ch.tutteli.atrium.translations.DescriptionBasic.IS
 import ch.tutteli.atrium.translations.DescriptionOptionalAssertion.EMPTY
+import ch.tutteli.atrium.translations.DescriptionResultAssertion
 import java.util.*
 
 fun <T : Optional<*>> _isEmpty(expect: Expect<T>): Assertion =
     ExpectImpl.builder.createDescriptive(expect, IS, RawString.create(EMPTY)) { !it.isPresent }
+
+fun <E, T : Optional<E>> _isPresent(expect: Expect<T>): ExtractedFeaturePostStep<T, E> =
+    ExpectImpl.feature.extractor(expect)
```

Comment:
The code is fine but the corresponding report in case of an error is not yet correct. See hint in the Spec

Reply:
I think I'm getting closer. The report produced by the test made more sense now. I'm still unsure about the Description

Reply:
Use `.withDescription("get()")` for the description